mslib/mscolab/conf.py

```diff
@@ -57,8 +57,20 @@ class default_mscolab_settings:
 
     DATA_DIR = os.path.join(BASE_DIR, "colabdata")
 
-    # mscolab data directory
-    MSCOLAB_DATA_DIR = os.path.join(DATA_DIR, 'filedata')
+    # mscolab data directory for operation git repositories
+    FILE_DATA = os.path.join(DATA_DIR, 'filedata')
+
+    # SSO by SAML2 is optional
+
+    # dir where mscolab single sign-on process files are stored
+    SSO_DIR = os.path.join(DATA_DIR, 'datasso')
+
+    # Enable SSL certificate verification during SSO between MSColab and IdP
+    SSO_SSL_CERT_VERIFICATION = True
+
+    # path to xmlsec1 (sign, verify, encrypt and decrypt XML documents)
+    # this is only needed for a testing IDP setup
+    # SSO_XMLSEC_PATH = os.path.join(os.environ["CONDA_PREFIX"], "bin", "xmlsec1")
```

Comment:
I don't think making this configurable is a good idea either. There is a standard environment variable to communicate the search path for executables, it's called `PATH`. Why not just use that?

Reply:
looks like it is fixed
```
>>> from saml2.sigver import get_xmlsec_binary
>>> get_xmlsec_binary()
'/home/reimar/Miniforge/envs/mssdev/bin/xmlsec1'
```